src/main/java/com/blackhearth/blockchain/node/BlockChainNodeData.java

```diff
@@ -8,4 +8,9 @@
 public class BlockChainNodeData {
     private int port;
     private String ip;
+
+    @Override
+    public String toString() {
+        return String.format("BCN{%s:%d}", ip, port);
```

Comment:
Tak na przyszłość
https://projectlombok.org/features/ToString

Reply:
Znam, ale nie drukowało to tak jak chcialem. Nie poprawiam